charts/athens-proxy/templates/deployment.yaml

```diff
@@ -12,7 +12,6 @@ spec:
   selector:
     matchLabels:
       app: {{ template "fullname" . }}
-      chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
```

Comment:
@rabadin I don't recall the reason we put this in to begin with. to be conservative, is it worth leaving in the chart name?

Reply:
We could leave the chart name, yes. As long as it doesn't change.
That being said, looking at a bunch of charts from `git@github.com:helm/charts.git` it seems the practice is to have the selector use `app` and `release` so I suggest following this pattern and remove `chart` altogether like I'd doing here.

Reply:
@arschles You okay with the removal of the `chart` bit in the selectors?

Reply:
>@arschles You okay with the removal of the chart bit in the selectors?

@rabadin yes, that's ok with me. thanks for the explanation!